packages/workbox-webpack-plugin/index.js

```diff
@@ -62,6 +62,25 @@ class WorkboxBuildWebpackPlugin {
       config.globPatterns = ['**/*.{html,js,css}'];
     }
 
+    if (compilation.options.output.publicPath && !config.manifestTransforms) {
+      const publicPath = compilation.options.output.publicPath;
+      const compiledAssets = [];
+      for (let key in compilation.assets) {
+        if (compilation.assets.hasOwnProperty(key)) {
+          compiledAssets.push(path.resolve(publicPath, './' + key));
```

Comment:
Don't need the `./` here. It's unnecessary because path.resolve, url.resolve and path.join will all treat `key` like its a directory anyway.